src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml.cpp

```diff
@@ -569,10 +593,18 @@ void GraphingCalculator::GraphSettingsButton_Click(Object ^ sender, RoutedEventA
 
 void GraphingCalculator::DisplayGraphSettings()
 {
-    auto graphSettings = ref new GraphingSettings();
-    graphSettings->SetGrapher(this->GraphingControl);
+    if (m_graphSettingsVM == nullptr)
```

Comment:
I would prefer GraphSettings to continue to create its ViewModel and keep it private.

To sync IsMatchAppTheme, we should instead create a dependency property IsMatchAppTheme in GraphingSettings use a two-way binding created code behind to sync values